docs/tutorials/control/air-quality-fleet.md

```diff
@@ -20,766 +20,718 @@ cost: 200
 
-In our example you could create a dashboard for Antonia with an API key to see the data from her location, and create a separate dashboard for RobotsRUs with a different API key to access the data from their location.
-If RobotsRUs wanted to separate their dashboards by sub-locations, you could set up API keys for RobotsRUs to access data for each of their sub-locations separately, or you could modify the example code to filter data by location name.
+document.cookie = "<SECRET COOKIE INFO>; path=/";
+document.cookie = "machinesWhoseCredentialsAreStored=<MACHINE ID>; path=/";
 
-You can then either deploy each dashboard on a web server you manage, or add a web server on one machine per customer that hosts the dashboard for the respective customer so that they can access their data on their local network.
-We leave this step to the reader.
+console.log("Set 2 cookies on localhost");
+```
 
-The following instructions describe how to set up an API key for one location.
+{{% /tablestep %}}
+{{% tablestep number=4 %}}
 
-1.  Create another file inside the <file>aqi-dashboard</file> folder and name it <file>main.ts</file>.
-    Paste the following code into <file>main.ts</file>:
+Run the following command to serve the app you are building:
 
-    ```typescript {class="line-numbers linkable-line-numbers"}
-    // Air quality dashboard
+```sh {class="command-line" data-prompt="$" data-output="2-10"}
+npm run start
+```
 
-    import * as VIAM from "@viamrobotics/sdk";
-    import { BSON } from "bson";
+Open the app in your browser at `http://127.0.0.1:8000/`.
 
-    async function main() {
-      const opts: VIAM.ViamClientOptions = {
-        credential: {
-          type: "api-key",
-          // Key with location operator permissions
-          // Replace <API-KEY> (including angle brackets)
-          payload: "<API-KEY>",
-          // Replace <API-KEY-ID> (including angle brackets)
-          authEntity: "<API-KEY-ID>",
-        },
-      };
+Then, open developer tools, go to the console and paste the copied JavaScript code to set your cookies.
 
-      const orgID: string = "<ORGANIZATION ID>"; // Replace
-      const locationID: string = "<LOCATION ID>"; // Replace
+{{% /tablestep %}}
+{{< /table >}}
 
-      // <Insert data client and query code here in later steps>
+### Add functionality to your code
 
-      // <Insert HTML block code here in later steps>
+Now that you have the connection code, you are ready to add code that establishes a connection from the computer running the code to the Viam Cloud where the air quality sensor data is stored.
+
+{{< table >}}
+{{% tablestep number=1 %}}
+
+You'll first create a client to obtain the organization and location ID. Then you'll get a `dataClient` instance which accesses all the data in your location, and then query this data to get only the data tagged with the `air-quality` tag you applied with your data service configuration.
```

Comment:
```suggestion
You'll first create a client to obtain the organization and location ID.
Then you'll get a `dataClient` instance which accesses all the data in your location, and then query this data to get only the data tagged with the `air-quality` tag you applied with your data service configuration.
```
[nit] newline
Also is "get" the right word to use for what you do with dataclient? Should it be "establish" or "instantiate" or something more precise, or is that too pedantic?